src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -137,6 +137,54 @@ public boolean LIC7(List<Point> points, int kPts, double length1) throws Illegal
         return false;
     }
 
+    /**
+     * LIC #14 checks if there exists at least one set of three data points separated by exactly <b>ePts</b> and <b>fPts</b>
+     * consecutive intervening points, respectively, that are the vertices of a triangle with area greater
+     * than <b>area1</b>, AND if there exist three data points separated by exactly <b>ePts</b> and <b>fPts</b> consecutive
+     * intervening points, respectively, that are the vertices of a triangle with area less than
+     * <b>area2</b>. The condition is not met when <b>points</b> has less than 5 elements.
+     *
+     * @param points list of radar echos ({@link Point})
+     * @param ePts first consecutive intervening points
+     * @param fPts second consecutive intervening points
+     * @param area1 the area of the first triangle
+     * @param area2 the area of the second triangle
+     * @return true if at least one set of three data points separated by exactly <b>ePts</b> and <b>fPts</b>
+     * consecutive intervening points, respectively forms a triangle with an area greater than <b>area1</b>,
+     * and if at least one set of three data points separated by exactly <b>ePts</b> and <b>fPts</b>
+     * consecutive intervening points, respectively forms a triangle with an area less than <b>area2</b>,
+     * false otherwise.
+     * @throws IllegalArgumentException is thrown if <b>points</b> is null, or if the length
+     *                                  of <b>points</b> - 3 is less than <b>ePts</b> + <b>fPts</b>.
+     */
+    public boolean LIC14(List<Point> points, int ePts, int fPts, double area1, double area2) throws IllegalArgumentException {
+        if (points == null) {
+            throw new IllegalArgumentException("Points list cannot be null");
+        }
```

Comment:
Don't forget to add a check that checks if the number of points is less than 5. Because if it is, then the condition is not met (i.e. LIC10 should return `false`). This is in the specification.

![bild](https://user-images.githubusercontent.com/70259607/151665445-a4b5f44c-27f0-464d-bef9-8c1dcccdf088.png)

Reply:
This should be fixed now.